spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafProperties.java

```diff
@@ -97,6 +100,11 @@
 	private String[] excludedViewNames;
 
 	/**
+	 * Template to use for the root view of an application.
+	 */
+	private String rootView = DEFAULT_ROOT_VIEW;
```

Comment:
No, this doesn't really need to be configurable. I'll remove it. I'm not that familiar with git to know how to undo the changes to this one file. Can you cherry-pick which files you pull?